### DISCUSSION TITLE
How to show data type image or varbinary in dbeaver

### DISCUSSION THREAD

**Discussion by Frank0321** [dbeaver/dbeaver#22706] [Q&A] [answered]
### Description

when my column type is  image or varbinary,
I can't see data in this column.
How can I do to review data in this column ?

### DBeaver Version

Version 23.3.1.202312241705

**Comment by E1izabeth** [comment]
Can't you open the Value Panel and see the selected cell value in a different representation?

![image](https://github.com/dbeaver/dbeaver/assets/28875055/2a6b154a-1d3d-46e9-a70b-7824824e1e79)

Here is a video about Value Panel with explanations https://www.youtube.com/watch?v=hylFIEF1W-Q

**Comment by Frank0321** [comment]
when I open  the Value Panel ....
Do I need to setting something ?
<img width="1134" alt="截圖 2024-01-04 下午3 36 06" src="https://github.com/dbeaver/dbeaver/assets/61242521/ecf5b744-6c14-41a6-a752-ee7a3a9ffc9f">

**Comment by E1izabeth** [comment]
Do you have Image option here?
![image](https://github.com/dbeaver/dbeaver/assets/28875055/2f657acf-eda5-49c7-81af-723f49148944)

**Comment by Frank0321** [comment]
I don't have this option

<img width="263" alt="截圖 2024-01-04 下午3 53 59" src="https://github.com/dbeaver/dbeaver/assets/61242521/f12cbdac-b2bc-4d19-b158-5290fd0b951a">

**Comment by LonwoLonwo** [comment]
Hello @Frank0321 
Please provide table DDL and data input for tests.

**Comment by Frank0321** [comment]
I  use spring session, and he create  SPRING_SESSION, and SPRING_SESSION_ATTRIBUTES two table

```
CREATETABLE SPRING_SESSION (
    PRIMARY_IDCHAR(36)NOTNULL,
    SESSION_IDCHAR(36)NOTNULL,
    CREATION_TIME BIGINTNOTNULL,
    LAST_ACCESS_TIME BIGINTNOTNULL,
    MAX_INACTIVE_INTERVALINTNOTNULL,
    EXPIRY_TIME BIGINTNOTNULL,
    PRINCIPAL_NAMEVARCHAR(100),
CONSTRAINT SPRING_SESSION_PKPRIMARYKEY (PRIMARY_ID)
);

CREATEUNIQUEINDEX SPRING_SESSION_IX1ON SPRING_SESSION (SESSION_ID);
CREATEINDEX SPRING_SESSION_IX2ON SPRING_SESSION (EXPIRY_TIME);
CREATEINDEX SPRING_SESSION_IX3ON SPRING_SESSION (PRINCIPAL_NAME);

CREATETABLE SPRING_SESSION_ATTRIBUTES (
    SESSION_PRIMARY_IDCHAR(36)NOTNULL,
    ATTRIBUTE_NAMEVARCHAR(200)NOTNULL,
    ATTRIBUTE_BYTES IMAGENOTNULL,
CONSTRAINT SPRING_SESSION_ATTRIBUTES_PKPRIMARYKEY (SESSION_PRIMARY_ID, ATTRIBUTE_NAME),
CONSTRAINT SPRING_SESSION_ATTRIBUTES_FKFOREIGNKEY (SESSION_PRIMARY_ID)REFERENCES SPRING_SESSION(PRIMARY_ID)ONDELETE CASCADE
);
```

and I save something to session, it can create data like this
```
INSERT INTO dbo.SPRING_SESSION_ATTRIBUTES (SESSION_PRIMARY_ID,ATTRIBUTE_NAME,ATTRIBUTE_BYTES) VALUES
	 (N'f486aafa-fc4d-497a-b6a2-00a83e0b9844',N'test',0xACED0005740006363034363234);

```

**Comment by Matvey16** [comment, marked as answer]
@Frank0321 Is the data in this cell actually an image? DBeaver won't give you an image option if the BLOB is not an imege like in this example.

https://github.com/dbeaver/dbeaver/assets/82543000/f3f5b672-d59d-43c8-948f-5ebc9ae393a2
